main/php_streams.h

```diff
@@ -282,8 +282,8 @@ END_EXTERN_C()
 		return; \
 	} \
 } while (0)
-#define php_stream_from_res_no_verify(xstr, pzval)	(xstr) = (php_stream*)zend_fetch_resource2((res), "stream", php_file_le_stream(), php_file_le_pstream())
-#define php_stream_from_zval_no_verify(xstr, pzval)	(xstr) = (php_stream*)zend_fetch_resource2_ex((pzval), "stream", php_file_le_stream(), php_file_le_pstream())
+#define php_stream_from_res_no_verify(xstr, pzval)	(xstr) = (php_stream*)zend_fetch_resource2((res), NULL, php_file_le_stream(), php_file_le_pstream())
```

Comment:
I don't understand the motivation, the no_verify just means that the macro won't return, so it's just an unclear name.
Also this behaviour change does not seem worth it for extensions?

Reply:
The motivation is to have an API that extension can rely on to get a `php_stream` (mainly from a zval) to ease the migration from resource to objects. And we have a bunch of case which use `zend_fetch_resource2` with `NULL` so they can handle the errors themselves.

I guess I'll just add a new macro/inline function

Reply:
I would prefer not doing those incremental changes. Once there is such migration, users will need to handle it anyway.